README.md

```diff
@@ -51,3 +56,8 @@ Once inside the container, you can install NiMARE:
 ```
 python /home/neuro/code/NiMARE/setup.py develop
 ```
+
+## Contributing
+
+Please see our [contributing guidelines](https://github.com/neurostuff/NiMARE/blob/master/CONTRIBUTING.md) for more information on contributing
```

Comment:
Could you also link the Code of Conduct here ?

Reply:
Absolutely. Thanks for taking a look and good catch!